src/main/java/com/work/management/service/log/impl/LoggingServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package com.work.management.service.log.impl;
+
+import com.google.common.collect.ImmutableMap;
+import com.work.management.service.log.LoggingService;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+final class LoggingServiceImpl implements LoggingService {
+
+  private static final Logger logger = LoggerFactory.getLogger(LoggingServiceImpl.class);
+
+  @Override
+  public void logRequest(HttpServletRequest httpServletRequest, Object body) {
+    StringBuilder stringBuilder = new StringBuilder();
```

Comment:
nit: rename stringBuilder to something more logical like messageToLog or requestMessage or requestLog.